packages/core/src/helpers/space/space.scss

```diff
@@ -1,11 +1,18 @@
-$allowed-fractionals: (1.5, 0.5, 0.25, 0.125);
-
+$allowed-floats: (1.5, 0.5, 0.25, 0.125);
 $base: 8px;
 
-@function space($number) {
-  @if $number % 1 != 0 and not index($allowed-fractionals, $number) {
-    @error '$number must be an integer or one of (#{$allowed-fractionals})';
+@function space($multiplier) {
+  @if not is-allowed($multiplier) {
+    @error '$number must be an integer or one of (#{$allowed-floats})';
```

Comment:
Either:

```suggestion
    @error '$multiplier must be an integer or (#{$allowed-floats})';
```

or

```suggestion
    @error '$multiplier must be an integer or (#{$allowed-floats}). Got: #{$multiplier}';
```

Reply:
Ah missed that, used editor name refactor feature, thanks!